src/main/java/jenkins/plugins/gerrit/workflow/GerritReviewStep.java

```diff
@@ -60,16 +60,22 @@ protected Execution(@Nonnull StepContext context) throws IOException, Interrupte
 
     @Override
     protected Void run() throws Exception {
+      String credentialsId = envVars.get("GERRIT_CREDENTIALS_ID");
+      String gerritApiUrl = envVars.get("GERRIT_API_URL");
       String branch = envVars.get("BRANCH_NAME");
+
+      if (gerritApiUrl == null) {
+        echo("GERRIT_API_URL is not available, disabling Gerrit integration");
+        return null;
+      }
+
```

Comment:
This part is repeated twice, but this isn't your fault but mine :-(
I rushed the implementation because I needed to present at the JenkinsWorld 2018 in Nice ... but I had in my TODO to refactor the code and make a common base class of GerritActions.

Reply:
most of the step is duplicated, I did not want to touch that at this point.